### PR TITLE
Add RelationshipClass.SCANS

### PR DESCRIPTION
**Pull request by aiwilliams**

### CODE REVIEW

File: src/RelationshipClass.ts [hunk 1]
```diff
@@ -73,8 +105,13 @@ export enum RelationshipClass {
...
   /**
-   * A relationship between any Entity and a utility Entity.
-   * The target entity can be "used" by many source entities.
+   * A relationship indicating an Entity utilizes another Entity. Typically, the
+   * target Entity can be "used" by many source Entities.
+   *
+   * Examples:
+   *
+   * * Host - USES -> DataStore
+   * * UserRole - USES -> AccessPolicy
```
Comment (aiwilliams) [inline src/RelationshipClass.ts:114]: I'd like see our relationship class docs move in this direction. What do you think?

File: src/RelationshipClass.ts [hunk 2]
```diff
@@ -43,7 +43,11 @@ export enum RelationshipClass {
   EXTENDS = 'EXTENDS',
 
   /**
-   * A relationship between any Entity and {Finding, Incident}.
+   * A relationship indicating an Entity is responsible for generating another Entity.
+   *
+   * Examples:
+   *
+   * * Project - GENERATED -> Finding
```
Comment (ndowmon) [inline src/RelationshipClass.ts:50]: Just so we don't lose any existing documentation

```suggestion
   * * Project - GENERATED -> {Finding, Incident}
```

**Review by ndowmon — APPROVED** [review]
Looks good, one small suggestion. I approved either way.

Is this a suggested relationship class to use for the relationship I just requested for Rapid7 (https://github.com/JupiterOne/graph-rapid7/pull/1#discussion_r497765485), or is that a coincidence?

> Can you also add an `insightvm_scan--MONITORS-->insightvm_asset` relationship?

File: src/RelationshipClass.ts (hunk 2 above)
Comment (aiwilliams) [reply src/RelationshipClass.ts:50]: I need to find an example of that relationship, couldn't see one in GitHub, perhaps my search-fu is off in all the context switching 😬

**Comment by aiwilliams** [comment]
> Is this a suggested relationship class to use for the relationship I just requested for Rapid7 ([JupiterOne/graph-rapid7#1 (comment)](https://github.com/JupiterOne/graph-rapid7/pull/1#discussion_r497765485)), or is that a coincidence?
> 
> > Can you also add an `insightvm_scan--MONITORS-->insightvm_asset` relationship?

I hadn't seen that yet, but yes, I made that mistake in the qualys integration (used `MONITORS` for a thing that `SCANS` a thing). Sounds like we'll have to address it in rapid7. How serendipitous! A coincidence!

**Pull request merged**